.nuspec/Microsoft.Maui.Controls.SingleProject.props

```diff
@@ -63,4 +63,45 @@
     />
   </ItemGroup>
 
+   <!-- iOS -->
+  <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' and '$(SingleProject)' == 'true' and '$(TargetPlatformIdentifier)' == 'ios' and '$(iOSProjectFolder)' != '' ">
+    <None Include="$(iOSProjectFolder)Info.plist" LogicalName="Info.plist" />
+  </ItemGroup>
```

Comment:
This is already above, do we need this?

https://github.com/dotnet/maui/blob/ad8980a1fc9e892ddf980b3da731002c5100dce1/.nuspec/Microsoft.Maui.Controls.SingleProject.props#L19-L20